js/index.js

```diff
@@ -0,0 +1,196 @@
+import { $, $all } from './util.js';
+
+// 오늘 날짜 문자열 반환
+const getTodayDate = () => {
+  const today = new Date();
+  const year = today.getFullYear();
+  const month = (today.getMonth() + 1).toString().padStart(2, '0');
+  const date = today.getDate().toString().padStart(2, '0');
+
+  const dayToString = ['일', '월', '화', '수', '목', '금', '토'];
+  const day = dayToString[today.getDay()];
+
+  return `${year}년 ${month}월 ${date}일 ${day}요일`;
+};
+
+// 할 일 아이템 생성
+const createItem = (text, isDone = false) => {
+  const newItem = document.createElement('li');
+
+  // draggable 속성 추가
+  newItem.setAttribute('draggable', true);
+  newItem.classList.add('draggable');
+
+  // 동그라미 아이콘
+  const circleIcon = document.createElement('i');
+  circleIcon.classList.add('fa-regular', 'fa-circle', 'cursor-pointer');
+
+  if (isDone) {
+    circleIcon.classList.add('fa-check-circle');
+  }
+  // 누르면 done으로 이동하는 이벤트 리스너 등록
+  circleIcon.addEventListener('click', moveItem);
+  newItem.append(circleIcon);
+
+  // 입력받은 Text
+  const todoText = document.createTextNode(text);
+  newItem.append(todoText);
+
+  // 쓰레기통 아이콘
+  const trashIcon = document.createElement('i');
+  trashIcon.classList.add('fa-solid', 'fa-trash-can', 'cursor-pointer');
+
+  // 누르면 삭제
+  trashIcon.addEventListener('click', deleteItem);
+  newItem.append(trashIcon);
+
+  return newItem;
+};
+
+// 할 일 추가 이벤트 핸들러
+const addItem = (e) => {
+  e.preventDefault();
+
+  // 빈 값 입력시 에러 처리
+  const inputValue = $('.input').value.trim();
+  if (!inputValue) {
+    $('.error').textContent = '내용을 입력해주세요';
+    return;
+  }
+  $('.error').textContent = '';
```

Comment:
이부분은
```suggestion
  const inputValue = document.querySelector('.input');
  if (!inputValue.value.trim()) {
    alert = ('내용을 입력해주세요');
  }
```
로 하면 더 깔끔할거 같아요!

Reply:
앗 다시 확인해보니 error을 쓰신 이유가 있었네요! alert 부분은 무시해 주셔도 됩니다! 개인적으로는 inputValue와 같은 변수에는 하나의 클래스만 선택하도록 하는 것이 또 쓰일 경우를 대비해서 더 좋더라구요. if문안에도 직관적인 내용이 들어왔을 때가 유지 보수하기 좋았던 것 같은데 이부분은 개인 취향이 아닐까.. 생각합니다!